### PR TITLE
feat: add docs for specifying taa accept mechanism

### PR DESCRIPTION
**Pull request by morrieinmaas**
Signed-off-by: Moriarty <moritz@animo.id>

### CODE REVIEW

File: guides/tutorials/agent-config.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ accepted.
 **Type**: `string`
 
 ```typescript title="example"
-endpoint: ["https://example.org:3000"]
+endpoint: ["https://example.org:3000"];
```
Comment (TimoGlastra) [inline guides/tutorials/agent-config.md:117]: these changes seem unrelated.

@blu3beri I also ran into problems with formatting. Can you look at adding a prettier config so we avoid these formatting changes?

**Review by TimoGlastra — CHANGES_REQUESTED** [review]
Looks good but a lot of unrelated formatting changes

File: guides/tutorials/agent-config.md (hunk 1 above)
Comment (morrieinmaas) [reply guides/tutorials/agent-config.md:117]: ah yes!

File: guides/tutorials/agent-config.md [hunk 2]
```diff
@@ -190,6 +194,12 @@ a JSON object like the
 
 Stringified JSON object of the transaction.
 
+### `indyLedgers.transactionAuthorAgreement`
+
+**Type**: `TransactionAuthorAgreement`
+
+JSON representation specifying the version and acceptance mechanism.
```
Comment (blu3beri) [inline guides/tutorials/agent-config.md:201]: Could you explain what the fields inside this object mean? In the same structure as the other fields

**Review by blu3beri — CHANGES_REQUESTED** [review]

**Review by TimoGlastra — APPROVED** [review]

File: guides/tutorials/agent-config.md [hunk 3]
```diff
@@ -190,6 +194,12 @@ a JSON object like the
 
 Stringified JSON object of the transaction.
 
+### `indyLedgers.transactionAuthorAgreement`
+
+**Type**: `TransactionAuthorAgreement`
+
+JSON representation specifying the version and acceptance mechanism. The version is the unique version of the transaction author agreement acceptance mechanism list (AML). The acceptance mechanism refers to the acceptance mechanism label of the item in the AML. For more details you may consult the [indy-node docs on AML](https://github.com/hyperledger/indy-node/blob/master/docs/source/transactions.md#transaction_author_agreement_aml)
```
Comment (blu3beri) [inline guides/tutorials/agent-config.md:201]: Its a good description, but I ment more in the style of like the walletConfig where the keys are seperately explained.

Like here: 

https://animo.github.io/aries-javascript-docs/guides/tutorials/agent-config#walletconfigid

**Review by blu3beri — CHANGES_REQUESTED** [review]

File: guides/tutorials/agent-config.md [hunk 4]
```diff
@@ -200,6 +200,18 @@ Stringified JSON object of the transaction.
 
 JSON representation specifying the version and acceptance mechanism. The version is the unique version of the transaction author agreement acceptance mechanism list (AML). The acceptance mechanism refers to the acceptance mechanism label of the item in the AML. For more details you may consult the [indy-node docs on AML](https://github.com/hyperledger/indy-node/blob/master/docs/source/transactions.md#transaction_author_agreement_aml)
 
+### `indyLedgers.transactionAuthorAgreement.version`
```
Comment (TimoGlastra) [inline guides/tutorials/agent-config.md:203]: Should this have four `####`? To indicate the extra level of nesting

**Comment by TimoGlastra** [comment]
@morrieinmaas @blu3beri any updates on this PR? Do we need to change what I raised in my comment? If not, can we merge this?

File: guides/tutorials/agent-config.md (hunk 4 above)
Comment (blu3beri) [reply guides/tutorials/agent-config.md:203]: Yeah I think that that would be best.

Comment (TimoGlastra) [reply guides/tutorials/agent-config.md:203]: @morrieinmaas back to you :)
